common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -218,6 +218,17 @@ class InheritanceMixin(XBlockMixin):
         default=False
     )
 
+    self_paced = Boolean(
+        display_name=_('Self Paced'),
+        help=_(
+            'Set this to "true" to mark this course as self-paced. Self-paced courses do not have '
+            'due dates for assignments, and students can progress through the course at any rate before '
```

Comment:
@schenedx I've intentionally left this the same as the [course field](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/course_module.py#L744-L753) from which it's taken. If this field were visible in Studio's advanced settings, it'd be helpful to mention valid values for the field in the help text.